.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - 6
```

Comment:
Add a comment that we're supporting 6 so that we can use AWS Lambda. Add latest stable ("node") as well. I'd like to use as much of the latest features as we can while still supporting AWS lambda.